sensor/utils/utils.go

```diff
@@ -19,6 +19,9 @@ func GetSensorKubernetesLabels() map[string]string {
 		KubernetesLabelName:      "stackrox",
 	}
 }
+func TLSSecretLabels() map[string]string {
+	return map[string]string{"rhacs.central.com/tls": "true"}
+}
```

Comment:
```suggestion
func GetTLSSecretLabels() map[string]string {
        labels := GetSensorKubernetesLabels()
        labels["rhacs.redhat.com/tls"] = "true"
	return labels
}
```

This should also include the regular labels. And I'd name it with `Get...` so it's consistent with the other functions.